File: HorseRacing.Contracts/Models/Game/Requests/GetAvailableSuit/GetAvailableSuitRequest.cs
using HorseRacing.Contracts.Base.Dto;
using HorseRacing.Contracts.Base.Requests;

namespace HorseRacing.Contracts.Models.Game.Requests.GetAvailableSuit
{
    public class GetAvailableSuitRequest : BaseRequest<BaseModelDto>
    {
        public GetAvailableSuitRequest() : base() { }
        public GetAvailableSuitRequest(BaseModelDto data) : base(data) { }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/requests/get-available-suit/get-available-suit-request.ts
import type { BaseModelDto } from "~/interfaces/api/contracts/base/dto/base-model-dto";
import type { BaseRequest } from "~/interfaces/api/contracts/base/requests/base-request";

export class GetAvailableSuitRequest implements BaseRequest<BaseModelDto> {
    Data: BaseModelDto;

    constructor(data?: BaseModelDto) {
            this.Data = data || { Id: ''};
        }
}
